### ISSUE TITLE
Add marionette cookbook recipes to the www site

### ISSUE THREAD

**Issue by joshbedo** [marionettejs/marionettejs.com#150]
I think it would be very beneficial to add recipes that have been created in the [Marionette-cookbook](https://github.com/MarionetteLabs/marionette-cookbook) to the *www* site. We could potentially create an examples sidebar that shows a few different *flexible* solutions for common problems aiming to be a "This is one way to do it but not the only way" solution for new users. I see a lot of commonly asked questions for UI problems asked on gitter and think this would be a great area to help provide solutions. 

We could provide a *live* working example with a source snippet/gist below. I also think this would encourage others to contribute different solutions that have worked for them on their projects and ultimately crowd source UI behaviors or features requested by the community. Thoughts?

**Comment by jasonLaster** [comment]
Yes! But the cookbooks need lots of presentation work to be good. Also you'll see they could get better too m
